src/config.json

```diff
@@ -103,7 +103,7 @@
         "google_ad_conversion": "kR9OCLas4JgBEOy2pucC"
     },
     "comm_wait_timeout": 600000,
-    "config": "narrative-refactor",
+    "config": "dev",
```

Comment:
deliberate?

Reply:
Oh. Forgot about that.

Uh, semi-deliberate. It doesn't affect anything except for a dev downloading the module. Running locally against narrative-refactor doesn't seem to work for me.